### DISCUSSION TITLE
[GitHub Incident] Deployment is not triggered by commits to main branch

### DISCUSSION THREAD

**Discussion by igorlanko** [vercel/community#3840] [Help] [answered]
### Summary

No deployment is being triggered even though there're > 5 commits since the last deployment.

Seems like it's possibly related to this today's issue https://www.vercel-status.com/incidents/r7md342b3d39? But it has the `resolved` status, and I'm still experiencing this.

### Steps to Reproduce

1. Push a commit to the main branch.
2. Expect Vercel to trigger the deployment.
3. Vercel is not showing the latest commit and therefore not triggering any new deployments.

**Comment by swarnava** [comment, marked as answer]
Hi,

It looks like GitHub is facing an outage which could be the reason you are facing this issue: https://www.githubstatus.com/incidents/f7lz4nc72bvp

**Reply by igorlanko** [reply]
Thanks, @swarnava! That's what I thought. Funnily, it just got resolved on my end too. Sorry to bother!

**Reply by swarnava** [reply]
You're very welcome! :)
